docs/sphinx/developers/examples/ExampleSuite.java

```diff
@@ -53,13 +53,16 @@ public static void main(String[] args) throws Exception {
 
     // Retrieve local test files
     URL resource =  ExampleSuite.class.getResource("test.fake");
+    URL overlappedResource =  ExampleSuite.class.getResource("test&sizeX=1024&sizeY=1024.fake");
```

Comment:
This file needs to be created (with `touch` or similar) under `docs/sphinx/examples` to prevent a NPE.